models/classes/LtiProvider/Validation/ValidationRegistry.php

```diff
@@ -31,22 +31,22 @@ class ValidationRegistry extends ConfigurableService
 
     private const VALIDATORS = [
         '1.1' => [
-            DataStore::PROPERTY_OAUTH_KEY => [['notEmpty']],
-            DataStore::PROPERTY_OAUTH_SECRET => [['notEmpty']],
-            RdfLtiProviderRepository::LTI_VERSION => [['notEmpty']],
+            DataStore::PROPERTY_OAUTH_KEY => [['NotEmpty']],
```

Comment:
i don't like the fact that we are using the name provided by this and not `ClassName::class` but i can live with it